README.md

```diff
@@ -47,15 +47,20 @@ Prebuilt binaries are available to download at https://www.alluxio.io/download .
 Download and start an Alluxio master and a worker. More details can be found in [documentation](https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Docker.html).
 
 ```bash
-# launch a master
-$ docker run -d --net=host\
-    -v /mnt/data:/opt/alluxio/underFSStorage\
+# Launch the Alluxio master
+$ docker run -d \
+    -p 19999:19999 \
+    --net=alluxio_nw \
+    --name=alluxio-master \
+    -v ufs:/opt/alluxio/underFSStorage \
     alluxio/alluxio master
-# launch a worker
-$ docker run -d --net=host --shm-size=1G\
-    -e ALLUXIO_WORKER_MEMORY_SIZE=1G\
-    -v /mnt/data:/opt/alluxio/underFSStorage\
-    -e ALLUXIO_MASTER_HOSTNAME=localhost\
+# Launch the Alluxio worker
+$ docker run -d \
+    --net=alluxio_nw \
+    --name=alluxio-worker \
+    --shm-size=1G \
```

Comment:
Should we use an environment variable here and in the `ALLUXIO_JAVA_OPTS` section so that users know they should be equal values?

Reply:
do you mean an env variable to represent 1G?

Reply:
Yes